### ISSUE TITLE
Unable to set the keyboard focus to the "General" tab using shift+ tab from the Font size edit box.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#302]
### Pre-requisites:
Launch the Visual Studio for Mac and Sign-in with Enterprise account.

Create projects:
1. Go to Visual Studio-> File-> New Solution. -> Multiplatform-> App-> Xamarin.Forms-> Blank Forms App.
2. Hit on Next button to create the project.
3. Hit on Debug(Play)/Build button to build/deploy the solution/project.
4. As per the choice/requirements, Change the Device platform like iOS/ Android and iPhone X iOS 11.3 etc.
5. Hit on Inspector icon button (i.e., in the top toolbar buttons of the VS) to inspect the project.
6. On hitting on inspector button, Xamarin Inspector is Launched.

### Repro-Steps:
1. Follow the Pre-requisites to Launch the Xamarin Inspector from Visual Studio.
2. Navigate to Xamarin Inspector menu, which is at the top of the Xamarin-Inspector window window.
3. Navigate to the Xamarin Inspector-> Preferences-> Hit on it.
4. Navigate to General tab of the General window.
5. Traverse the all the interactive elements of the General Window using Tab Key.
6. Do the Same traversing in the reverse order.
7. Once keyboard focus is on the Font Size text box, Hit on shift+tab key to traverse in the reverse order.
8. Observe that the keyboard is focus in reverse diarection from the Font size edit box.
### Expected Behavior
Keyboard user should be able to get focus on "General" Tab using shift+tab key from Font size edit box.
### Actual Behavior
Focus reamains at Font size edit box, even though user is hitting on the Shift+ tab key. 
Note: Although no issue is observed in the forward navigation.
### User Impact
User has to go to "General" Tab only by forward navigation which wastes user time. Although its very low user impact bug.

![kb1_pref_generaltab_shifttaborderloss](https://user-images.githubusercontent.com/22956130/39995495-ff09800a-5799-11e8-9815-07b925f47fc2.png)

### Environment
Visual Studio for Mac Enterprise [Version: 7.5 Preview (7.5 build 1222)]
Xamarin Inspector [Version: 1.4 (master/b3f92f9)]

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Inspector-MAC
Wipro
MAS2.1.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1830
